test/api/fragment.js

```diff
@@ -45,7 +44,7 @@ describe("API: JSDOM.fragment()", () => {
     assert.strictEqual(frag.firstChild.textContent, "Hi");
   });
 
-  it("should respect ignore any options passed in", () => {
+  it("should ignore any options passed in", () => {
```

Comment:
> Also while you're there if you could fix the test title from "should respect ignore any options passed in" to "should respect the relevant options passed in", that'd be ideal.

Changed the name to ignore instead of respect since this API doesn't allow any options.